client/src/pages/Feed.js

```diff
@@ -418,7 +418,7 @@ const Feed = (props) => {
     if (!searchKeywords || !searchKeywords.length) return handleSearchClear();
     dispatchAction(SET_VALUE, "searchKeyword", searchKeywords);
     handleSearchSubmit(searchCategory);
-  }, [handleSearchClear, handleSearchSubmit, searchCategory, searchKeywords]);
```

Comment:
Does anyone have an idea why does this gets added every commit, it breaks everything by creating an infinite loop.
How do I make it ignore this line?

Reply:
@Zaydme I think one of them is an object? For a quick answer, you can delete dependencies one by one to see which one produce an infinite loop. And you can ignore that like below, and then commit. Then maybe you should consider what you lose when you ignore a dependency.

 ` }, [handleSearchClear, handleSearchSubmit, searchCategory]); // eslint-disable-line react-hooks/exhaustive-deps`